server/resources/resource_management_test.go

```diff
@@ -1292,6 +1305,27 @@ var _ = Describe("Resource manager", func() {
 					Expect(ok).To(BeTrue())
 					Expect(theResource).To(HaveKeyWithValue("test_string", "Ia, ia, HJPEV fhtang!"))
 				})
+
+				It("Should only modify updated fields", func() {
+					networkSchema, _ := manager.Schema("network")
+
+					err := resources.CreateResource(context, testDB, fakeIdentity, networkSchema, adminNetworkData)
```

Comment:
a bit shorter version that reads as a sentence:
`Expect(resources.CreateResource(...)).To(Succeed())`